dhctl/cmd/dhctl/commands/bootstrap/bootstrap.go

```diff
@@ -18,6 +18,8 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"github.com/deckhouse/deckhouse/dhctl/pkg/operations/bootstrap"
```

Comment:
It looks like imports ain't sorted. How did our linter miss it?

Reply:
I'ill see it